android-test/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,6 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="okio.android.test">
+  <application
```

Comment:
Do you need an `android:networkSecurityConfig="@xml/network_security_config"` attribute here? Also, what's the reasoning behind allow clear text traffic here but forbidding in the `network_security_config`, is this supposed to set different rules for different API versions?

Reply:
To be honest it’s just ’cause I copy-pasted OkHttp’s AndroidManifest.xml, and it seemed like keeping this would avoid problems around raw TCP sockets. This isn’t a regular application so opting out of security policy seemed natural.